src/test/java/resources/voucherManagementTest.feature

```diff
@@ -66,3 +66,13 @@ Feature: Voucher Management Api Test
     Given I can create an negative VoucherRequestDTO for voucher creation
     When I call the create voucher API with expected status of 400 and stub "/createVoucher"
     Then I should be able to assert the create voucher validation for negative response
+
+  Scenario: VR-003 Error Validity check for Redeem Voucher API for negative request body
+    Given I can create an negative RedeemVoucherRequestDTO to redeem a voucher
+    When I call the redeem voucher API with expected status of 400
+    And I should be able to assert the redeem voucher validation for negative response
```

Comment:
there is a slight diff with the requirement provided provided

Reply:
Yes, as Redeem Voucher API is synchronous API. We need to check the agentId in the request body synchronously.